markdown/en/ff-suggest.md

```diff
@@ -34,11 +34,15 @@ The `ff-suggest` element respects only section child elements. The section eleme
 
 ## Adding a Suggest container
 
-You need to annotate an element with the `data-container="suggestType"` attribute for each suggest type configured in the FACT-Finder backend.
+You need to annotate an element with the `data-container="suggestType"`
+attribute for each suggest type configured in the FACT-Finder backend.
 
-If no suggestions are found for this type the `ff-suggest` element hides the annotated container.
+If no suggestions are found for this type the `ff-suggest` element hides
```

Comment:
Can I have a comma here, please? ^^
`If no suggestions are found for this type, the ...`